forch/local_state_collector.py

```diff
@@ -79,17 +80,20 @@ def _check_process_info(self):
 
         # fill up process info
         for target_name in self._target_procs:
+            varz_name = target_name + '_process_state'
```

Comment:
as per other comment, this should be done using labels, not as a different metric name

Reply:
Addressed.